README.md

```diff
@@ -58,7 +58,3 @@ To register a device, run as root from LmP device:
 
 **NOTE** If you're emulating the device with QEMU, by default the gateway (host) IP is `10.0.2.2`.
 (i.e. `sudo DEVICE_API=http://10.0.2.2:80/sign lmp-device-register [-T <TOKEN>] [-f <factory>]`)
-
-# Testing/Troubleshooting
-This project includes a simple `fake-lmp-device-register` that can be used
-to quickly test this project's API.
```

Comment:
I think this small note is good to have.
You could also change it to reference `contrib/fake-lmp-device-register`, and squash with the previous commit.

Reply:
It is done what was suggested.
Since it has the LGTM from you and Andy as the approvals I am getting this one merged.